src/main/resources/application-search.yml

```diff
@@ -26,3 +26,12 @@ mod-linked-data:
   kafka:
     topic:
       bibframe-index: search.bibframe
+
+folio:
```

Comment:
There is a problem with having the configuration in `application-search.yml` file.
For integrating with Inventory, we need to create `inventory.instance_ingress` topic. As this topic is not related to "search" functionality, we will be adding the configuration for `inventory.instance_ingress` in `application.yml` file.

That will cause problems as spring will not merge the list of topics from 2 yaml files.

May be get rid of "search" profile and use just "folio" profile?